src/wacks4square/square.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import os
+import re
+from datetime import datetime, timedelta
+from typing import List, Optional
+
+import requests
+from dotenv import load_dotenv
+
+from wacksbywarby.constants import SQUARE_TIME_FORMAT
+from wacksbywarby.models import Sale
+
+logger = logging.getLogger("square")
+
+# prod
+BASE_API = "https://connect.squareup.com/v2"
+
+MAIN_LOCATION_ID = "LF63C50H5VTEK"
+BACKUP_LOCATION_ID = "LHTYTJXMD1TBW"
```

Comment:
I wonder if we should keep these in the .env?

Reply:
yeah i was thinking we could move these out too. it's a little more work to deploy but i guess that's fine